src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
```

Comment:
ColorSupplier must be a class variable
```suggestion
        Color color = сolorSupplier.getRandomColor();
```